website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -1649,6 +1656,34 @@ var Range = ace.require('ace/range').Range;
                 buttonRow.appendChild(button);
                 return button;
             };
+            var makeHelpButton = function(id,title,XShift){
+              var button = document.createElement("li");
+              //Based off the search button classes
+              //<button type=button class="btn osf-search-btn" data-toggle="modal" data-target="#search-help-modal"><i class="fa fa-question fa-lg"></i></button>

```

Comment:
Probably we don't need this explanation text. The reason we added it to the other PR was because we were pulling from something done externally, and it was something that was based on logic that's not necessarily obvious. In this case, you're just doing something like we do it elsewhere on the site, so you don't need to document it.

Our dev guide might help to decide when to comment, but generally rarely. https://cosdev.readthedocs.io/en/latest/style_guides/python.html#on-comments